src/main/java/de/maxhenkel/voicechat/voice/common/Utils.java

```diff
@@ -121,18 +121,10 @@ public static short bytesToShort(byte b1, byte b2) {
         } else if (rot > 0.5F) {
             perc = 0.5F - (rot - 0.5F);
         }
+        perc = perc * (1 - dif);
 
-        float left = Math.max(0.3F, perc < 0F ? Math.abs(perc * 2F) : 0F);
-        float right = Math.max(0.3F, perc >= 0F ? perc * 2F : 0F);
-
-        float fill;
-        if (left > right) {
-            fill = 1F - left;
-        } else {
-            fill = 1F - right;
-        }
-        left += fill;
-        right += fill;
+        float left = perc < 0F ? Math.abs(perc * 1.4F) + 0.3F : 0.3F;
+        float right = perc >= 0F ? (perc * 1.4F) + 0.3F : 0.3F;
```

Comment:
As perc is between 0 and +- 0.5 these numbers normalise it between 0.3 and 1.

Reply:
the 1.4 moves it between 0-0.7 and then the +0.3 offsets it to 0.3 and 1

Reply:
I have to reopen this conversation because I ran some numbers and noticed that this code creates left = 0.3F and right = 0.3F if perc is 0 (sound ahead). However the old code created left = 1 and right = 1 for this case. Correct me if I'm wrong but I think the old fill code had it's reason for existance.

Reply:
I think if you add this after your code it just works :todd:

float fill = 1F - Math.max(left, right);
left += fill;
right += fill;

Reply:
Ah yep, got the logic backwards where it's 0.3 straight ahead and increases the close side. instead of both 1 straight ahead and decreases the far side. Couldn't out my finger on why it sounded weird but this was it thankyou.